Libs/RaptureAddins/MongoDb/src/main/java/rapture/blob/mongodb/MongoDBBlobStore.java

```diff
@@ -24,22 +24,25 @@
 package rapture.blob.mongodb;
 
 import java.io.InputStream;
+import java.net.HttpURLConnection;
 import java.util.Map;
 
 import org.apache.commons.lang.StringUtils;
 
+import com.mongodb.MongoException;
+
 import rapture.blob.BaseBlobStore;
 import rapture.common.CallingContext;
+import rapture.common.Messages;
 import rapture.common.RaptureURI;
 import rapture.common.exception.RaptureExceptionFactory;
 
-import com.mongodb.MongoException;
-
 public class MongoDBBlobStore extends BaseBlobStore {
     private final static String INSTANCE_NAME = "default";
     public static final String PRIMARY_CONFIG = "prefix";
     public static final String SECONDARY_CONFIG = "grid";
     public static final String MULTIPART = "multipart";
+	private Messages mongoMsgCatalog = new Messages("Mongo");
```

Comment:
formatting is off, should be 4 spaces not 8.  applies to most changes in this PR.